apps/rush-lib/src/logic/PackageChangeAnalyzer.ts

```diff
@@ -71,17 +76,30 @@ export class PackageChangeAnalyzer {
     return projectState;
   }
 
-  private _getData(): Map<string, Map<string, string>> | undefined {
+  private async _getData(terminal: Terminal): Promise<Map<string, Map<string, string>> | undefined> {
     const repoDeps: Map<string, string> | undefined = this._getRepoDeps();
     if (!repoDeps) {
       return undefined;
     }
 
     const projectHashDeps: Map<string, Map<string, string>> = new Map<string, Map<string, string>>();
 
-    // pre-populate the map with the projects from the config
+    const ignorePatternsByProject: Map<string, IMinimatch[]> = new Map<string, IMinimatch[]>();
+
+    // pre-populate the map with the projects from the config, and load all project configurations up-front
     for (const project of this._rushConfiguration.projects) {
       projectHashDeps.set(project.packageName, new Map<string, string>());
+
+      const projectConfiguration:
+        | RushProjectConfiguration
+        | undefined = await RushProjectConfiguration.tryLoadForProjectAsync(project, undefined, terminal);
```

Comment:
@iclanton Can the `rush-project.json` settings be inherited from a rig?  That seems useful

Reply:
I can confirm that they can today, I have it working in our local monorepo with 5.43.0.  🎉